src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/TransfersControllerTests/WhenIViewTheTransferConnectionInvitationAuthorizationComponent.cs

```diff
@@ -31,7 +33,7 @@ public void Arrange()
             _mediator = new Mock<IMediator>();
             _mediator.Setup(m => m.SendAsync(_query)).ReturnsAsync(_response);
 
-            _controller = new TransfersController(null, _mapper, _mediator.Object);
+            _controller = new TransfersController(null, _mapper, _mediator.Object,_configuration);
```

Comment:
spacing

Reply:
No longer an issue as transfersController doesn't need the extra parameter